### DISCUSSION TITLE
Question on how to customize the p2p-media-loader-demo

### DISCUSSION THREAD

**Discussion by soq2000** [Novage/p2p-media-loader#200] [Q&A] [answered]
Dear All,
Thank you for a very nice project on hybrid transmission server-client and peer2peer.
I cannot find your support forum therefore I have to post here for your support. Thank you for your understanding.
I would like to ask you several questions as the below
-Whenever  I click on the button "Create new peer", I expect to see a new node in the below graphic connected to the blue circle named "You". It signifies that a new node just connects to the initial peer instead of the origin server. But I cannot see such situation. Sometimes I see a node 75.74.151.161 connecting to node "You". And sometimes there are several "Unknown" nodes connecting to the node "You" as well. (I am almost sure that my newly created node is not among these unknown nodes, because  when I turn my new node off, the unknown nodes are still there) 
-Can I replace the source Video with my own streaming RTSP server? I tried to change the URL in vain. It is a black screen
-How can I control the switching process between peer and server in a more explicit way, meaning I can switch manually ?
Thank you in advance for you guide
Best regard

**Comment by mrlika** [comment, marked as answer]
Please check our new v1 release.

> How can I control the switching process between peer and server in a more explicit way, meaning I can switch manually 
Now you can change configuration in the runtime that allows to turn on and off P2P

>. Can I replace the source Video with my own streaming RTSP server
P2P Media Loader supports only chunked video: HLS and MPEG-DASH. RTSP is not supported

> Whenever I click on the button "Create new peer", I expect to see a new node in the below graphic connected
It works like BitTorrent - not always two peers can connect
